assets/quick-add-bulk.js

```diff
@@ -52,6 +57,18 @@ class QuickAddBulk extends HTMLElement {
     });
   }
 
+  resetQuantityInput(id) {
+    const input = document.getElementById(id);
+    input.value = input.getAttribute('value');
+    this.isEnterPressed = false;
+  }
+
+  cleanErrorMessageOnType(event) {
+    event.target.addEventListener('keypress', () => {
+      event.target.setCustomValidity('');
+    }, { once: true });
+  }
```

Comment:
I added this function to remove the tooltip as soon as you start typing another qty. Otherwise, the tooltip will keep being open and blinking after each type until you submit a new qty. 
That how it looked without cleaning the error message on type 👇

![12-28-72hik-irzsv](https://github.com/Shopify/dawn/assets/105315663/28fde53f-71a7-43d9-86ce-1cc248f9d583)

Reply:
That is before experience 👆